torchmetrics/functional/text/bleu.py

```diff
@@ -122,8 +122,8 @@ def _bleu_score_compute(
 
 
 def bleu_score(
-    reference_corpus: Sequence[Sequence[Sequence[str]]],
-    translate_corpus: Sequence[Sequence[str]],
+    reference_corpus: Sequence[Sequence[str]],
+    translate_corpus: Sequence[str],
```

Comment:
Maybe, can we allow a user to pass in a single `translate_sentence`, i.e. as a string, and then evaluate against `reference_corpus` of a type `Sequence[str]`? Similar to https://github.com/mjpost/sacrebleu/blob/master/sacrebleu/metrics/bleu.py#L400

cc: @SkafteNicki @Borda

Reply:
hmm, that sounds good :]

Reply:
Do you mean we want to support both `translate_corpus` and `translate_sentence` in the function? As in, supporting for both a single sentence and multiple translate sentences? 

 Or alternatively, update the function where it supports single translate sentence instead?

cc @stancld ?

Reply:
Something like:

```
translate_corpus: Union[Sequence[str], str]
...
translate_corpus = [translate_corpus] if isinstance(translate_corpus, str) else translate_corpus
```

Reply:
@mrleu sorry for a late response. I would be for the suggested option with:
```
translate_corpus: Union[str, Sequence[str]
reference_corpus: Union[Sequence[str], Sequence[Sequence[str]]]
```

Reply:
```py
translate_corpus: Union[str, Sequence[str]
 reference_corpus: Sequence[Union[str, Sequence[str]]
```

Reply:
No worries. Got it!